test/configs.test.js

```diff
@@ -63,17 +63,24 @@ describe('Testing  Configuration files', () => {
         expect(validated.details).toEqual('API version is either v1 or v2');
     });
     
-    it('4.5) configs validate must check the type of values of configs entered by the user, vaultAppName part', () => {
+    it('4.5) configs validate must check the type of values of configs entered by the user, vaultLoginUrl part', () => {
         let myConfigs = new configsObject(inValidTestConfigs(4));
         let validated = myConfigs.validateConfigs();
         expect(validated.valid).toBe(false);
-        expect(validated.details).toEqual('vaultAppName must be string');
+        expect(validated.details).toEqual('vaultLoginUrl must be string');
     });
 
-    it('4.5) configs validate must check the type of values of configs entered by the user, followAllRedirects part', () => {
+    it('4.6) configs validate must check the type of values of configs entered by the user, vaultAppName part', () => {
         let myConfigs = new configsObject(inValidTestConfigs(5));
         let validated = myConfigs.validateConfigs();
         expect(validated.valid).toBe(false);
```

Comment:
Can you please revert this as it was before, just update the test number as its duplicated, the second 4.5 to be 4.6, and add 2 more new unit tests for the vaultLoginUrl, one for that success and the other that fails.